lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -15,33 +19,48 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name))
+        .map((tile) => _buildGridTile(withName: tile.name, path: tile.path))
         .toList();
 
     return Center(
-        child: GridView.count(
-      primary: false,
-      padding: const EdgeInsets.all(20),
-      crossAxisSpacing: 10,
-      mainAxisSpacing: 10,
-      crossAxisCount: 2,
-      children: gridTile,
-    ));
+      child: GridView.count(
+        primary: false,
+        padding: const EdgeInsets.all(Spacing.spacingMD),
+        crossAxisSpacing: 20,
+        mainAxisSpacing: 10,
```

Comment:
Use Spacing constants here too or ignore it if you need to reconstruct this view with the design established.